ui/src/config/section/network.js

```diff
@@ -29,8 +29,20 @@ export default {
       icon: 'apartment',
       permission: ['listNetworks'],
       resourceType: 'Network',
-      columns: ['name', 'state', 'type', 'vpcname', 'cidr', 'ip6cidr', 'broadcasturi', 'domain', 'account', 'zonename'],
-      details: ['name', 'id', 'description', 'type', 'traffictype', 'vpcid', 'vlan', 'broadcasturi', 'cidr', 'ip6cidr', 'netmask', 'gateway', 'aclname', 'ispersistent', 'restartrequired', 'reservediprange', 'redundantrouter', 'networkdomain', 'zonename', 'account', 'domain'],
+      columns: () => {
+        var fields = ['name', 'state', 'type', 'vpcname', 'cidr', 'ip6cidr', 'broadcasturi', 'domain', 'account', 'zonename']
+        if (!['Admin'].includes(store.getters.userInfo.roletype)) {
```

Comment:
may be, this can be moved to a util method _isAdmin()_

```suggestion
        if (!isAdmin()) {
```

Reply:
makes sense but
```
$ grep -r isAdmin\ \(\)\ \{ ui/src/*
ui/src/views/compute/KubernetesServiceTab.vue:    isAdmin () {
ui/src/views/network/CreateNetwork.vue:    isAdmin () {
ui/src/views/network/UpdateNetwork.vue:    isAdmin () {
ui/src/views/network/CreateIsolatedNetworkForm.vue:    isAdmin () {
ui/src/views/network/CreateL2NetworkForm.vue:    isAdmin () {
ui/src/views/network/CreateSharedNetworkForm.vue:    isAdmin () {
ui/src/views/offering/AddComputeOffering.vue:    isAdmin () {
ui/src/views/offering/AddDiskOffering.vue:    isAdmin () {
ui/src/views/offering/AddNetworkOffering.vue:    isAdmin () {
ui/src/views/offering/AddVpcOffering.vue:    isAdmin () {
ui/src/views/offering/UpdateOfferingAccess.vue:    isAdmin () {
```
I think it should also be unified and made available in a generic way, than.